employee-rostering-frontend/docker/default.conf

```diff
@@ -0,0 +1,30 @@
+server {
+    listen       8080;
+    server_name  localhost;
+
+    #charset koi8-r;
```

Comment:
I copied this conf file from the nginx base image. It contained a lot of commented out example code. I removed some of that (e.g. proxy example because we already use that piece of configuration) but I thought other examples might be useful in the future so I've left them in place.

I know commented out code is 99.9% forbidden in regular source code but I think the examples might be relevant in this case. For example if we run into encoding issues and start debugging, seeing the example might save a lot of time.

I'm still open to removing all commented example code if you think it's the best way.

Reply:
Good point. I don't insist on removing the line. If there had been e.g. utf-8 I would have cared less. Just this unusual encoding drew my attention.